apps/meteor/server/startup/migrations/v269.ts

```diff
@@ -1,10 +1,13 @@
+import { MongoInternals } from 'meteor/mongo';
+
 import { addMigration } from '../../lib/migrations';
-import { Rooms } from '../../../app/models/server';
 
+// Remove Deprecated Omnichannel Queue Collection
 addMigration({
 	version: 269,
-	up() {
-		Rooms.tryDropIndex({ 'tokenpass.tokens.token': 1 });
-		Rooms.tryDropIndex({ tokenpass: 1 });
+	async up() {
```

Comment:
oops.. looks like this migration was moved to 270.. this one can be reverted 😬

Reply:
Revered the old migration :heavy_check_mark: